osm2gmns/func_lib/extract_relation_id.py

```diff
@@ -17,30 +17,66 @@
 class OSM_RelationID_Finder:
     """A class to find the osm relation id of a place of interest globally
        Use the Nominatim API to find the relation id of a place of interest
+
+    Args:
+        poi_name (str): the name of the place of interest,
+            e.g., "Arizona State University"
+            e.g., "Tempe, Arizona, USA"
+            e.g., "Arizona, USA"
+
+    Example:
+    >>> import osm2gmns as og
+    >>> rel = og.OSM_RelationID_Finder("Arizona State University")
```

Comment:
please use the CamelCase style for the naming of classes in osm2gmns.

Reply:
Thanks and i have changed to CamelCase style in the updated code.